share/templates.d/99-generic/runtime-install.tmpl

```diff
@@ -28,6 +28,8 @@ installpkg grubby
     ## https://bugzilla.redhat.com/show_bug.cgi?id=2011615
     ## bfa-firmware contains only obsolete files - see
     ## https://bugzilla.redhat.com/show_bug.cgi?id=2152202
+    ## qcom-firmware we pull in again lower down but *only* on aarch64, it is
+    ## no use on other arches - https://bugzilla.redhat.com/show_bug.cgi?id=2178852
```

Comment:
'not used' :)

Reply:
well, I can change it if you like, but "no use" is a common phrase meaning "not of any use"?

Reply:
Or 'of no use' :) But it doesn't look as goofy as it did pre-coffee so it's fine as is.